hwilib/devices/bitbox02.py

```diff
@@ -135,6 +133,40 @@ def attestation_check(self, result: bool) -> None:
             )
 
 
+def _parse_path(nstr: str) -> Sequence[int]:
```

Comment:
Related PR: #374

Reply:
Will include this after that PR is merged (can also be in a future PR).